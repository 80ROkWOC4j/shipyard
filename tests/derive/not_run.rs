use shipyard::*;

#[system(Test)]
fn test() {}

fn main() {}
