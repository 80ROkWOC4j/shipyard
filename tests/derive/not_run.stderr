error: Systems have only one method: run
 --> $DIR/not_run.rs:3:1
  |
3 | #[system(Test)]
  | ^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
